appcenter/android/build.gradle

```diff
@@ -18,8 +18,8 @@ android {
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
     compile 'com.facebook.react:react-native:+'
-    compile 'com.microsoft.appcenter:appcenter:1.7.0'
+    compile 'com.microsoft.appcenter:appcenter:1.8.0'
 
     compile project(':AppCenterReactNativeShared')   // For testing with TestApp
```

Comment:
As discussed, please revert to JCenter dependency as part of the RN release process. Same for the other 3 modules.